cmake/OpenEXRSetup.cmake

```diff
@@ -159,9 +159,11 @@ endif()
 
 option(OPENEXR_FORCE_INTERNAL_ZLIB "Force using an internal zlib" OFF)
 if (NOT OPENEXR_FORCE_INTERNAL_ZLIB)
-  find_package(ZLIB QUIET)
+  if(NOT TARGET ZLIB::ZLIB)
```

Comment:
It's surprising/not-surprising that find_package(ZLIB) doesn't know about aliases. One more corner case to think about when maintaining cmake scripts...

Reply:
Might be worth raising with CMake upstream. I don't know if there's an official policy for it, but it feels like the existence of the target should be checked because worst-case you'll redefine it.